recipe/meta.yaml

```diff
@@ -1,13 +1,14 @@
 {% set name = "vscode-jupyter" %}
-{% set version = "2020.12.1" %}
+{% set version = "2021.02.1" %}
+{% set install_version = "2021.2.0" %}
```

Comment:
Unfortunately the package version doesn't match the release tag:

https://github.com/microsoft/vscode-jupyter/blob/2021.02.1/package.json#L4